blt/blt.yml

```diff
@@ -33,10 +33,13 @@ modules:
     enable: {  }
```

Comment:
@blairlearn  It wont let me comment on code that wasn't changed but we should probably add these modules to the "uninstall" section of "local:" and "ci:". At some point you will start syncing databases instead of doing fresh installs and having SimpleSAML installed will break things.

Reply:
Seems reasonable, and done!